Sources/SwerverCore/Request/HttpRequest.swift

```diff
@@ -0,0 +1,34 @@
+import Foundation
+
+public class HttpRequest {
+    
+    private let method: String
+    private let url: String
+    private let httpVersion: String
+    
+    public init(method: String, url: String, httpVersion: String) {
+        self.method = method
+        self.url = url
+        self.httpVersion = httpVersion
+    }
+    
+    public func getMethod() -> String {
```

Comment:
Why return a String when you can return an enum? There are a finite number of methods that you need to worry about with an HTTP server.